tests/fast-integration/eventing-test/eventing-test-prep.js

```diff
@@ -147,6 +155,32 @@ describe('Eventing tests preparation', function() {
     await prepareAssetsWithCompassFlow();
   });
 
+  it('Prepare eventing-sink function', async function() {
+    debug('Preparing EventingSinkFunction');
+    await deployEventingSinkFunction();
+    await waitForEventingSinkFunction();
+  });
+
+  it('Eventing-sink function should be reachable through secured API Rule', async function() {
```

Comment:
we actually do not care in the scope of the eventing test
just exposing as some https endpoint without authentication should be enough

Reply:
I have removed auth for this.